HideSidebar.plugin.js

```diff
@@ -8,21 +8,24 @@
  * @source https://github.com/Pieloaf/BetterDiscordPlugins/blob/main/HideSidebar.plugin.js
  * @updateUrl https://raw.githubusercontent.com/Pieloaf/BetterDiscordPlugins/main/HideSidebar.plugin.js
  */
-module.exports = (_ => {
-
+module.exports = ((_) => {
     function createSelector(className) {
-        return '.' + className.replace(/ /g, '.')
+        return "." + className.replace(/ /g, ".");
     }
 
-    const sidebarSelector = createSelector(BdApi.findModuleByProps("hasNotice").sidebar)
-    const guildBarSelector = createSelector(BdApi.findModuleByProps("hasNotice").guilds)
-    const sidebarBtn = document.createElement('span');
+    const sidebarSelector = createSelector(
+        BdApi.findModuleByProps("guilds").sidebar
+    );
+    const guildBarSelector = createSelector(
+        BdApi.findModuleByProps("guilds").guilds
+    );
```

Comment:
main changes are here, the rest are auto-formatting changes (prettier didn't like what it saw).